packages/web/src/pages/upload-page/components/EditPageNew.tsx

```diff
@@ -47,53 +43,71 @@ const EditTrackSchema = Yup.object().shape({
 export const EditPageNew = (props: EditPageProps) => {
   const { tracks, setTracks, onContinue } = props
 
-  const [{ metadata: trackMetadata }] = tracks
-
-  const initialValues: EditFormValues = useMemo(
+  const initialValues: TrackEditFormValues = useMemo(
     () => ({
-      ...trackMetadata,
-      artwork: null,
-      description: '',
-      releaseDate: moment().startOf('day'),
-      tags: '',
-      licenseType: {
-        allowAttribution: null,
-        commercialUse: null,
-        derivativeWorks: null
-      }
+      trackMetadatas: tracks.map((track) => ({
+        ...track.metadata,
+        artwork: null,
+        description: '',
```

Comment:
so for example i added '' for description to ensure we dont get the "controlled/uncontrolled" warning in the console.